libraries/lib-wave-track/WaveClip.h

```diff
@@ -309,6 +318,20 @@ class WAVE_TRACK_API WaveClip final :
       */
    );
 
+   bool
+   GetFloatAtTime(double t, size_t iChannel, float& value, bool mayThrow) const;
+
+   void SetFloatsFromTime(
```

Comment:
Please comment that the new setting functions will succeed with out-of-bounds requests, only changing what is in bounds.

Reply:
`GetSamples` (not a new function) should also comment that `start` is relative to play start
`GetFloatAtTime` should comment that `t` is relative to play start